ciao-launcher/tests/examples/start_legacy_volume.yaml

```diff
@@ -1,10 +1,8 @@
 ---
 start:
-  requested_resources:
-     - type: vcpus
-       value: 2
-     - type: mem_mb
-       value: 370
+  requirements:
+    vcpus: 2
+    mem_mb: 256
```

Comment:
Was there a reason for changing this value?

Reply:
Just a copy and paste flail.